wordpress-mobile/WordPressMocks
# Update screenshot mocks

#### Proposed Changes

This updates the mocks for new iOS app store screenshots.

Significant changes:

* New sites introduced in the mocks (new primary site and two additional sites)
* Modified some mocks to return the same response regardless of site (where it doesn't matter for the tests or screenshots)

#### To Test

Ensure all UI tests pass and screenshots are generated with expected mock data on this PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/15055
